### PR TITLE
Fea 1178

### PR DESCRIPTION
**Pull request by owennicol**
#### What does this PR do? (please provide any background)
https://hxshortbreaks.atlassian.net/browse/FP-2263

This styles the more information page. 
@davidallardsb is doing some 'enhancements' within his jira scope, so this is a 'base' level styling.

#### How can this be tested?
`cd pattern-library && npm link ../brands`

#### Any tech debt?
Nope
#### Screenshots / Screencast
image

#### What gif best describes how you feel about this work?

- [ ] I have checked our general [contributing document](https://github.com/holidayextras/culture/blob/master/CONTRIBUTING.md) and the project specific [contributing document](../blob/master/CONTRIBUTING.md) (if present) and I'm happy for this to be reviewed.

---

**Reviewer**
- [ ] :+1:
- [ ] UXUI Team :+1:
- [ ] I don't think this PR needs any additional reviewers

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Ensured the release has the correct version bump.
- Checked all the work locally on pattern library.

### CODE REVIEW

File: brands/alton-towers/src/less/backgrounds.less [hunk 1]
```diff
@@ -41,11 +41,25 @@
...
-    text-decoration: underline;
+  color: @white;
+  .link-variant(fade(@bg-brand, 100%), darken(@white, 10%), @white);
+  h1 {
+    font-size:3em;
+    color: @white;
+    .fa-star {
+    font-size: 0.7em;
+    vertical-align: top;
+  }}
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/backgrounds.less]: double `}}`
new line?

File: brands/alton-towers/src/less/backgrounds.less [hunk 2]
```diff
@@ -41,11 +41,25 @@
 }
 .bg-brand {
   background-color: @bg-brand;
-  background-image: url(@bg-brand-image);
   .bg-cover;
   .bg-inverse;
-  a {
-    text-decoration: underline;
+  color: @white;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/backgrounds.less]: You could use `.bg-inverse` here

File: brands/alton-towers/src/less/backgrounds.less [hunk 3]
```diff
@@ -41,11 +41,25 @@
...
+  .link-variant(fade(@bg-brand, 100%), darken(@white, 10%), @white);
+  h1 {
+    font-size:3em;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/backgrounds.less]: there is a h1 font size variable, we should keep all h1's consistent

File: brands/alton-towers/src/less/backgrounds.less [hunk 4]
```diff
@@ -41,11 +41,25 @@
...
+    color: @white;
+    .fa-star {
+    font-size: 0.7em;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/backgrounds.less]: try `.small` or `@font-size small`

File: brands/alton-towers/src/less/backgrounds.less [hunk 5]
```diff
@@ -61,6 +75,7 @@
 }
 .bg-primary {
   background-color: @bg-primary;
+  color: @white;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/backgrounds.less]: may be able to use `.bg-inverse`

File: brands/alton-towers/src/less/backgrounds.less [hunk 6]
```diff
@@ -79,6 +94,7 @@
 }
 .bg-secondary {
   background-color: @bg-secondary;
+  color: @white;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/backgrounds.less]: again may be able to use `.bg-inverse` here

File: brands/alton-towers/src/less/navbar.less [hunk 7]
```diff
@@ -30,45 +29,53 @@
...
     background-color: @brand-tertiary-light;
     color: @brand-primary;
   }
 }
 
-.navbar-default .navbar-nav > li.active > a {
-  font-weight: 700;
+.navbar-default .navbar-nav>li>a {
+  transition: background-color .25s ease, color .25s ease;
+  color: @brand-primary;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/navbar.less]: Navbar link colours are a variable - https://github.com/twbs/bootstrap/blob/v3.3.7/less/variables.less#L372

File: brands/alton-towers/src/less/navbar.less [hunk 8]
```diff
@@ -30,45 +29,53 @@
...
+  &:hover, &:focus{
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/navbar.less]: can use variables - https://github.com/twbs/bootstrap/blob/v3.3.7/less/variables.less#L372

File: brands/alton-towers/src/less/navbar.less [hunk 9]
```diff
@@ -30,45 +29,53 @@
...
-
   .navbar-inverse {
-    .navbar-collapse .navbar-nav > li,
-    .navbar-nav .open .dropdown-menu > li {
-      > a {
-        color:@navbar-default-link-color;
-        &:hover, &:active, &:focus {
-          color:@navbar-default-link-hover-color;
-          background:@navbar-default-link-hover-bg;
+    .navbar-collapse .navbar-nav>li,
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/navbar.less]: few variable you can change here - https://github.com/twbs/bootstrap/blob/v3.3.7/less/variables.less#L230

File: brands/alton-towers/src/less/progress-trackers.less [hunk 10]
```diff
@@ -1 +1,14 @@
 @import '../../../brand-globals/src/less/progress-trackers';
+.progress-tracker {
+  .navbar-inverse;
+  font-weight: normal;
+  >li {
+    color: @text-color;
+    >a:not(.btn):not(.list-group-item):not(.panel-title):not(.alert-link):not([role=tab]) {
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/progress-trackers.less]: this is specific in that its inside `.progress-tracker` so surely you don't need this level of specificity as a progress tracker wont contain buttons, alerts, etc

though i have not tested this

File: brands/alton-towers/src/less/backgrounds.less (hunk 2 above)
Comment (owennicol) [reply brands/alton-towers/src/less/backgrounds.less]: If you look at the line above - it's there just ineffective

File: brands/alton-towers/src/less/backgrounds.less (hunk 3 above)
Comment (owennicol) [reply brands/alton-towers/src/less/backgrounds.less]: Due to the font, we have to make this nice and big for this section, but I don't think we want it so shouty for every other section where the h1 appears.

File: brands/alton-towers/src/less/backgrounds.less (hunk 4 above)
Comment (owennicol) [reply brands/alton-towers/src/less/backgrounds.less]: needed to be @font-size-medium ;)

File: brands/alton-towers/src/less/backgrounds.less (hunk 6 above)
Comment (owennicol) [reply brands/alton-towers/src/less/backgrounds.less]: looks like this needs the text colour here

File: brands/alton-towers/src/less/navbar.less (hunk 9 above)
Comment (owennicol) [reply brands/alton-towers/src/less/navbar.less]: Not sure I understand this one

File: brands/alton-towers/src/less/progress-trackers.less (hunk 10 above)
Comment (owennicol) [reply brands/alton-towers/src/less/progress-trackers.less]: Yes, yes we do. :(

File: brands/alton-towers/src/less/navbar.less [hunk 11]
```diff
@@ -1,16 +1,15 @@
...
 .navbar-collapse,
 .navbar {
-  border:0;
+  border: 0;
 }
 
-.navbar-primary{
+.navbar-primary {
   margin-bottom: @navbar-margin-bottom;
+  .navbar-inverse;
```
Comment (jodiedoubleday) [inline brands/alton-towers/src/less/navbar.less]: good use!

**Pull request merged**